src/util.js

```diff
@@ -12,5 +12,15 @@ module.exports = {
     return _.every(collection, function (item) {
       return _.isString(item);
     });
+  },
+
+  isArrayOfArrays: function (collection) {
+    return _.isArray(collection) && _.every(collection, function (item) {
+      return _.isArray(item);
+    });
+  },
```

Comment:
I wonder if there's a way we can re-use React's `propTypes` validators for stuff like this? Do they expose any validation functions? So we could use `React.PropTypes.arrayOf(React.PropTypes.array)` somehow.